ui/src/app/components/apps-panel.spec.tsx

```diff
@@ -266,17 +260,6 @@ describe(AppsPanel.name, () => {
     await expectExpandableApp(user, 'Jupyter');
   });
 
-  it('should disable apps when initial credit expiration is enabled and billing is inactive', async () => {
```

Comment:
"inactive billing" is no longer a valid Workspace state.  Billing validity is either unknown (when user-provided) or based on the _initial-credits_ state of valid, exhausted or expired.